domain/service/servicehealth_store.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2016 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"strings"
+	"errors"
+
+	"github.com/control-center/serviced/datastore"
+	"github.com/control-center/serviced/datastore/elastic"
+)
+
+func (s *storeImpl) GetServiceHealth(ctx datastore.Context, svcId string) (*ServiceHealth, error) {
+	defer ctx.Metrics().Stop(ctx.Metrics().Start("storeImpl.GetServiceHealth"))
```

Comment:
Please change `"storeImpl.GetServiceHealth"` to `"ServiceStore.GetServiceHealth"` to be more consistent with the other metrics on storeImpl